x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/trial_license_complete_tier/execution_logic/eql_alert_suppression.ts

```diff
@@ -255,6 +257,96 @@ export default ({ getService }: FtrProviderContext) => {
           );
         });
 
+        it('should deduplicate existing alert created without suppression', async () => {
```

Comment:
We should either define what "deduplicate" means for suppression, or (IMO) use more precise language. "Deduplicate" is confusing because it could be interpreted as either:

* Ignoring the alert entirely
* Suppress the alert by updating the existing alert's count/range

Maybe something like:
```suggestion
        it('does not count alerts created without suppression toward new suppression values', async () => {
```

Reply:
If you know that alert deduplication happens before suppression, then "deduplication of alert" here can only have one (logical) interpretation; I agree with that.

We should also be kind and leave context wherever we can. Even this description isn't quite correct, because the existing alert isn't being deduplicated, the new one is:

```suggestion
        it('deduplicates new alerts if they were previously created without suppression', async () => {
```

Reply:
agree, this test name looks better